src/common/Core/SearchResultItemActionUtility/createShowItemInFileExplorerAction.ts

```diff
@@ -13,4 +13,5 @@ export const createShowItemInFileExplorerAction = ({ filePath }: { filePath: str
     handlerId: "ShowItemInFileExplorer",
     fluentIcon: "DocumentFolderRegular",
     hideWindowAfterInvocation: true,
+    keyboardShortcut: "Ctrl+o",
```

Comment:
@tkohlmeier Probably we have to pass the keyboard shortcuts from the outside as on macOS the keyboard shortcut will be `Cmd+o` or maybe something completely different.